test/completion.js

```diff
@@ -385,96 +241,251 @@ describe('Completion', () => {
     })
   })
 
-  describe('getCompletion() - bash', () => {
-    it('returns default completion to callback', () => {
+  describe('bash', () => {
+    it('it returns a list of commands as completion suggestions', () => {
```

Comment:
nit, `it('returns a list of commands as completion suggestions')`, vs., `it('it`.

Reply:
Ah, will fix. :)

Reply:
Done!